<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player</class>
 <widget class="QMainWindow" name="player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>player</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,4">
      <item>
       <layout class="QVBoxLayout" name="Player">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="lblSong">
            <property name="text">
             <string>Canción:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblActualSong">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Reproductor">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnPrev">
              <property name="text">
               <string>Prev</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPlay">
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnNext">
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPause">
              <property name="text">
               <string>Pause</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStop">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="prgBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnMute">
              <property name="text">
               <string>Mute</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sldVolume">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="value">
               <number>20</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Add">
          <item>
           <layout class="QHBoxLayout" name="Name">
            <item>
             <widget class="QLabel" name="lblName">
              <property name="text">
               <string>Nombre:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Author">
            <item>
             <widget class="QLabel" name="lblAuthor">
              <property name="text">
               <string>Autor: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtAutor"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Select">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Dirección: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblAddress">
              <property name="text">
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSelect">
              <property name="text">
               <string>Seleccionar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="text">
             <string>Añadir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Table"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnReverse">
      <property name="text">
       <string>Invertir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReproductor_de_Audio">
    <property name="title">
     <string>Reproductor de Audio</string>
    </property>
   </widget>
   <addaction name="menuReproductor_de_Audio"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
